TgeeT/Questioner_v2
# JWT implementation

#### What does this PR do?

This PR is meant to secure endpoints with jwt
#### Description of Task to be completed?

add jwt tokens to secure activities

#### How should this be manually tested?
How should this be manually tested?

follow the steps below:
1. git clone this repository [here](https://github.com/TgeeT/Questioner_v2.git) 
2. Create a virtual environment and activate it.
3. git checkout to ft-jwt-163279143 branch
4. Run the application and try to post a meetup without login

### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163279143

### Screenshots
![image](https://user-images.githubusercontent.com/45405036/51644441-3e6bb880-1f81-11e9-9978-e134707d5780.png)
